src/panels/lovelace/cards/hui-area-card.ts

```diff
@@ -405,15 +412,20 @@ export class HuiAreaCard
     if (this._config.show_camera && "camera" in entitiesByDomain) {
       cameraEntityId = entitiesByDomain.camera[0].entity_id;
     }
+    cameraEntityId = "camera.demo_camera";
 
     const imageClass = area.picture || cameraEntityId;
+
+    const ignoreAspectRatio = this.layout === "grid";
```

Comment:
You want to ignore aspect ratio if it has an image class or layout is grid, so you can combine this check:
```suggestion
    const ignoreAspectRatio = imageClass || this.layout === "grid";
```